pynopegl-utils/pynopegl_utils/viewer/__init__.py

```diff
@@ -165,6 +166,8 @@ def _select_script(self):
         script = QUrl(script).path()  # handle the file:// automatically added by Qt/QML
 
         if script.endswith(".py"):
+            if platform.system() == "Windows" and script.startswith("/"):
+                script = script[1:]
```

Comment:
Too bad we can't use Python 3.9:
```python
if platform.system() == "Windows":
    script = script.removeprefix("/")
```

:(

Reply:
I guess when Apple decides to bump its default python version :)